virtualfish/virtual.fish

```diff
@@ -625,44 +625,107 @@ function __vf_help --description "Print VirtualFish usage information"
     echo "For full documentation, see: $help_url"
 end
 
-function __vf_globalpackages --description "Toggle global site packages"
-    set -l enabled
+function __vf_globalpackages --description "Manage global site packages"
+    argparse -si -n "vf globalpackages" "h/help" -- $argv
```

Comment:
One of the things I've been trying to do recently is to use the long version of flags — always, everywhere, in all projects. It's more readable, and that way folks have to look up what flags mean **way** less often. So here I suggest using the following instead:
```fish
argparse --stop-nonopt --ignore-unknown --name "vf globalpackages" "h/help" -- $argv
```
(I realize there are other places in the same file that pre-date this newer philosophy and that thus use short flags. I'll eventually change those as well, in due time.)